packages/webdriver-image-comparison/src/clientSideScripts/getBoundingClientRect.ts

```diff
@@ -0,0 +1,17 @@
+import type { ElementPosition } from './elementPosition.interfaces.js'
+
+/**
+ * Get the element position relative to the viewport
+ */
+export function getBoundingClientRect(
+    element: HTMLElement,
+): ElementPosition {
+    const { height, width, x, y } = element.getBoundingClientRect()
+
+    return {
+        height: Math.round(height),
```

Comment:
Why round this rather than returning the actual value?

Reply:
I' ve seen cases where these were floats and would "break" my implementation. You can check this by adding a float width and height to an element

```
$0.getBoundingClientRect()
DOMRect {x: 624, y: 234, width: 33.328125, height: 36, top: 234, …}bottom: 270height: 36left: 624right: 657.328125top: 234width: 33.328125x: 624y: 234[[Prototype]]: DOMRect
```